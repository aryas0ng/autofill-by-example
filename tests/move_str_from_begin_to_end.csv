"90005 Los Angeles, CA","Los Angeles, CA 90005"
"10032 New York, NY","New York, NY 10032"
"15557 Rockwood, PA","Rockwood, PA 15557"
"25840 Fayetteville, WV","Fayetteville, WV 25840"
"27014 Cooleemee, NC","Cooleemee, NC 27014"
"30314 Atlanta, GA",
"32189 Satsuma, FL",
"35139 Sayre, AL",
"37130 Murfreesboro, TN",
"38801 Tupelo, MS",
"40161 Rhodelia, KY",
"43066 Radnor, OH",
"34470 Ocala, FL",
"46256 Indianapolis, IN",
"48133 Erie, MI",
"50707 Evansdale, IA",
"53817 Patch Grove, WI",
"53924 Cazenovia, WI",
"56279 Pennock, MN",
"60910 Aroma Park, IL",
"64752 Hume, MO",
"67513 Alexander, KS",
"68357 Filley, NE",