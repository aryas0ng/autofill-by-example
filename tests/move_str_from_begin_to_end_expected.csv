"90005 Los Angeles, CA","Los Angeles, CA 90005"
"10032 New York, NY","New York, NY 10032"
"15557 Rockwood, PA","Rockwood, PA 15557"
"25840 Fayetteville, WV","Fayetteville, WV 25840"
"27014 Cooleemee, NC","Cooleemee, NC 27014"
"30314 Atlanta, GA","Atlanta, GA 30314"
"32189 Satsuma, FL","Satsuma, FL 32189"
"35139 Sayre, AL","Sayre, AL 35139"
"37130 Murfreesboro, TN","Murfreesboro, TN 37130"
"38801 Tupelo, MS","Tupelo, MS 38801"
"40161 Rhodelia, KY","Rhodelia, KY 40161"
"43066 Radnor, OH","Radnor, OH 43066"
"34470 Ocala, FL","Ocala, FL 34470"
"46256 Indianapolis, IN","Indianapolis, IN 46256"
"48133 Erie, MI","Erie, MI 48133"
"50707 Evansdale, IA","Evansdale, IA 50707"
"53817 Patch Grove, WI","Patch Grove, WI 53817"
"53924 Cazenovia, WI","Cazenovia, WI 53924"
"56279 Pennock, MN","Pennock, MN 56279"
"60910 Aroma Park, IL","Aroma Park, IL 60910"
"64752 Hume, MO","Hume, MO 64752"
"67513 Alexander, KS","Alexander, KS 67513"
"68357 Filley, NE","Filley, NE 68357"